extensions/colorslurp/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "colorslurp",
+  "title": "ColorSlurp",
+  "description": "Control ColorSlurp's color picker features from Raycast",
+  "icon": "colorslurp-icon.png",
+  "author": "Jordan-Ellis",
+  "categories": [
+    "Applications",
+    "Design Tools",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "show-magnifier",
```

Comment:
shouldn't this be `show-picker`? 🤔

Reply:
The Magnifier in ColorSlurp is what allows you to pick colors from your screen, so “Pick Color” is supposed to be a shortcut to showing the magnifier.  I could rename it to “Show Magnifier“, to be more inline with what it does if you’d like. 

I realized that the name of the other command pick-color didn’t align with the title of “Edit Color“, so I just pushed a change that fixes the naming.